### PR TITLE
Stateful v1.11

### PR DESCRIPTION
**Pull request by transmissions12**

### CODE REVIEW

File: eth/protocols/snap/sync.go [hunk 1]
```diff
@@ -87,8 +87,8 @@ const (
 	trienodeHealRateMeasurementImpact = 0.005
 
 	// minTrienodeHealThrottle is the minimum divisor for throttling trie node
-	// heal requests to avoid overloading the local node and excessively expanding
-	// the state trie breadth wise.
```
Comment (itsdevbear) [inline eth/protocols/snap/sync.go:91]: Typo

**Review by itsdevbear — CHANGES_REQUESTED** [review]
Lgtm just the minor fix

**Comment by itsdevbear** [comment]
@transmissions12 we don't merge this PR.

We just keep it open as a easy way to watch the diff.

File: eth/protocols/snap/sync.go (hunk 1 above)
Comment (transmissions12) [reply eth/protocols/snap/sync.go]: done

**Pull request closed without merging**